### PR TITLE
Authed Analytics 

### PR DESCRIPTION
**Pull request by JParisFerrer**
Analytics have auth. Students can see their own analytics, but not course analytics. Also both improves and worsens the navbar.

### CODE REVIEW

File: src/pages/courseInstanceAnalytics/courseInstanceAnalytics.ejs [hunk 1]
```diff
@@ -72,6 +72,10 @@
     },
     "data": {
       "sortOrder": "value-desc",
+      "smallSegmentGrouping": {
+			"enabled": true,
```
Comment (nwalters512) [inline src/pages/courseInstanceAnalytics/courseInstanceAnalytics.ejs:76]: tabs? pls :(

**Review by nwalters512 — APPROVED** [review]
there is code

File: src/pages/courseInstanceAnalytics/courseInstanceAnalytics.ejs (hunk 1 above)
Comment (JParisFerrer) [reply src/pages/courseInstanceAnalytics/courseInstanceAnalytics.ejs:76]: this wasn't me, xd

Comment (Binary-Eater) [reply src/pages/courseInstanceAnalytics/courseInstanceAnalytics.ejs:76]: Brackets? @PiotrGalusza

**Pull request merged**